client/app/widgets/ToggleLocale/index.jsx

```diff
@@ -11,7 +11,7 @@ export type Props = {
   locales: string[],
 };
 
-const options = (locales) => {
+const options = locales => {
```

Comment:
We should keep the parenthesis here, our settings in `yarn lint` will throw a warning otherwise (as seen in CircleCI).